compiler/rustc_resolve/src/late.rs

```diff
@@ -1200,7 +1200,7 @@ impl<'ra: 'ast, 'ast, 'tcx> Visitor<'ast> for LateResolutionVisitor<'_, 'ast, 'r
                 if let TyKind::Path(None, ref path) = ty.kind
                     // We cannot disambiguate multi-segment paths right now as that requires type
                     // checking.
-                    && path.is_potential_trivial_const_arg()
+                    && path.is_potential_trivial_const_arg(false)
```

Comment:
this.

I think making this use `self.features().min_generic_const_args()` is spooky :thinking: We would need to update logic furthur down to call methods that can actually resolve multi-seg paths and idk if such methods exist.

So yeah, tl;dr we just make both of these `false` and handle it down the road when we support unbraced const arguments of more complex paths (probably under a different feature gate)